packages/react-storage/src/components/StorageBrowser/actions/configs/types.ts

```diff
@@ -69,7 +69,12 @@ export interface ActionListItemConfig {
   /**
    * list item label
    */
-  label: string;
+  label?: string;
+
+  /**
+   * list item key (used for displayText)
+   */
+  key: string;
```

Comment:
No key at all, since we wouldn't be able to provide typing for it on the displayText API without complicated type inference. 

in the displayText API we add `getActionListItemLabel` in the location detail view interface, and then run the getter for each action list item with the `label` value provided from the configuration, our default implementation will look like:
```
getActionListItemLabel: (label) => label
```